src/interfaces/api/index.ts

```diff
@@ -0,0 +1,43 @@
+export * from './Search';
+export * from './Title';
+
+export enum ApiType {
```

Comment:
same here, I think that we won't need all of these endpoints

Reply:
Yeah maybe, but I think it would be better to have them just in case. Frontend won't be able to access those methods directly. They have only access to ApiProvider class that only has methods that we use.